client/Main.elm

```diff
@@ -1,17 +1,18 @@
-module Main exposing (..)
+module Main exposing (FromServer(..), FromUi(..), ItemId, Model, Msg(..), fromServer, init, main, update, view, viewItem)
 
+import Browser
+import Api exposing (..)
 import Dict exposing (..)
 import Html exposing (..)
-import Html exposing (program)
 import Html.Events exposing (..)
 import Http
-import Api exposing (..)
+import Debug
```

Comment:
Could you get rid of `Debug` since it is not allowed to be used in production code or published packages? In 0.19 there are only functions for numbers and `toString` has to be implemented by the user.

https://package.elm-lang.org/packages/elm/core/latest/Debug

Reply:
I'll fix this up later tonight. I was just getting something minimally working, and know `Debug` got deprecated (and a good thing too!).